german/Stopfen/ueberlegen/ueberlegen.md

```diff
@@ -0,0 +1,3 @@
+Sie entschlie�en sich, dem Anflug von Panik entgegenzutreten und erst einmal durchzuatmen. Dann schauen Sie sich im Boot genau um. Hier befinden sich keine Ruder oder sonstige Gegenst�nde.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.